src/maui/SSW.Rewards.Mobile/SSW.Rewards.Mobile/Pages/QuizDetailsPage.xaml

```diff
@@ -69,22 +72,22 @@
                                   CurrentItemChangedCommand="{Binding CurrentQuestionChangedCommand}"
                                   x:Name="QuestionsCarousel">
                         <CarouselView.ItemTemplate>
-                            <DataTemplate>
+                            <DataTemplate x:DataType="viewModels:QuizQuestionViewModel">
                                 <ScrollView HeightRequest="700">
                                     <StackLayout BindableLayout.ItemsSource="{Binding MyAnswers}"
                                                  RadioButtonGroup.GroupName="{Binding QuestionId}"
                                                  HorizontalOptions="FillAndExpand"
                                                  Padding="20"
                                                  Spacing="40">
                                         <BindableLayout.ItemTemplate>
-                                            <DataTemplate>
-                                                <RadioButton Value="{Binding QestionAnswerId}"
+                                            <DataTemplate x:DataType="viewModels:QuizAnswerViewModel">
+                                                <RadioButton Value="{Binding QuestionAnswerId}"
                                                              IsChecked="{Binding IsSelected}"
                                                              WidthRequest="300"
                                                              TextColor="White"
                                                              Content="{Binding Text}"
-                                                             ControlTemplate="{StaticResource RadioButtonTemplate}">
-                                                </RadioButton>
+                                                             ControlTemplate="{StaticResource {x:Null}}"/>
+                                                <!-- The ControlTemplate="{StaticResource RadioButtonTemplate}" is missing in the project. Was it lost at some point or it was a stub? -->
```

Comment:
Before migrating to MAUI `RadioButtonTemplate` lived in the `App.xaml`. Was it lost during migration or intentionally removed?

Reply:
#416 test comment